app/perfaddresscache.cpp

```diff
@@ -87,12 +87,6 @@ PerfAddressCache::SymbolCacheEntry PerfAddressCache::findSymbol(const QByteArray
     --it;
 
     if (it->offset <= relAddr && (it->offset + it->size > relAddr || (it->size == 0))) {
-        // demangle symbols on demand instead of demangling all symbols directly
```

Comment:
the commit message for this change has no relation whatsoever to this change. why are you removing this?

Reply:
where is the mangling handled then instead? where is that cached? as-is this patch looks totally wrong, is there maybe a change hunk in one of the other patches which changes the place where the mangling is done?

Reply:
Yes, it is used in the first commit - 61be877. I need demangled 'symname' but they are all already demangled.

Reply:
that's broken -  fold this patch into the first then and make sure that the demangled symbol is still cached